### DISCUSSION TITLE
How to process specific notes (or even know that they are processed)

### DISCUSSION THREAD

**Discussion by ghost** [brianpetro/obsidian-smart-connections#129] [Ideas]
Smart connection works well ... but I have no idea what notes it has processed and which it has not.
Also, I want it to answer questions using specific notes, but it tells me it does not know the note.
This all boils down to two questions:
- how do I know which notes are processed?
- how can I make sure that a note is processed?

**Comment by brianpetro** [comment]
Hi @ingewertwijn , welcome to the Smart Connection community!

This feature is still a work in progress, but you can look for the three little icons in the bottom right corner of messages that utilize your notes as context.

Clicking these will copy the context to your clipboard, and you can paste that into a note for review. You can read more about this in https://github.com/brianpetro/obsidian-smart-connections/issues/112#issuecomment-1485301523.

In short, this feature is still a work in progress, and getting better insights into the notes used as context will be coming soon!
